content/blog/2024/09/usr-merge/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Usr Merge"
+author: "evan"
+categories:
+  - news
+date: 2024-09-04
+featuredimage: "/usr-merge-banner-image.jpg"
+url: "/2024/09/04/usr-merge"
+---
+
+Back in May, 2024, we made our initial attempt at doing what we're calling "Usr Merge". While initially hopeful, it quickly became clear that there were significant problems with our approach, and that a new one would be needed. This post will go into what happened in May, and outline what we plan to do next time.
+
+## What is Usr Merge?
+
+The old way of organizing the Linux filesystem split files between `/bin` and `/usr/bin`, `/sbin` and `/usr/sbin`, `/lib` and `/usr/lib`, and `/lib64` and `/usr/lib64`. In recent years, there has been a push to unify everything by putting everything in the `/usr` file tree, e.g. `/usr/bin` and `/usr/lib64`, and adding compatibility symbolic links (symlinks) pointing to the directories in `/usr`. There are several reasons for this:
```

Comment:
> [...]unify everything by putting everything in [...]

word repetition, maybe "unify the structure by putting everything in..." or something like that?